great_expectations/data_context/data_context.py

```diff
@@ -941,6 +941,12 @@ def delete_expectation_suite(self, expectation_suite_name):
         Returns:
             True for Success and False for Failure.
         """
+        key = None
+        keys = self.stores[self.expectations_store_name].list_keys()
+        for item in keys:
+            sval = repr(item)
+            if expectation_suite_name.expectation_suite_name in sval:
+                key=item
```

Comment:
It appears that the issue was that key was never created. Rather than a round-trip through list_keys and using repr, consider the following implementation:
```suggestion
        key = ExpectationSuiteIdentifier(expectation_suite_name)
```

Is that addressing the same need?

Reply:
We get a split error bug on just that one line.